commands/help.py

```diff
@@ -1,45 +1,54 @@
-    _embedMessage.add_field(helpMessage, commandPrefix2 + "adminverify @Mention",
-                            "Manually verifies a user. Note that this does NOT add in a role and simply adds them to the database\nEx: " + commandPrefix2 + "adminverify " + client.user.mention,
-                            False)
-    _embedMessage.add_field(helpMessage, commandPrefix2 + "lockdown role",
-                            "Sets send message permissions to false for specified role\nEx: " + commandPrefix2 + "lockdown " + "Tron", False)
-    _embedMessage.add_field(helpMessage, commandPrefix2 + "removeduedate", "Remove's a due date\nEx: " + commandPrefix2 + "removeduedate", False)
     _embedMessage.add_field(helpMessage, commandPrefix2 + "removequote \"quote with spaces\" Name",
-                            "Removes a quote from the individual of your choice\nEx: " + commandPrefix2 + "addQuote \"Life is Good\", Bedi", False)
+                            "ADMIN ONLY - Removes a quote from the individual of your choice\nEx: " + commandPrefix2 + "addQuote \"Life is Good\", Bedi", False)
+
+    _embedMessage.add_field(helpMessage, commandPrefix2 + "lockdown role",
+                            "ADMIN ONLY - Sets send message permissions to false for specified role\nEx: " + commandPrefix2 + "lockdown " + "Tron", False)
+    _embedMessage.add_field(helpMessage, commandPrefix2 + "unlock role",
+                            "ADMIN ONLY - Sets send message permissions to True for specified role\nEx: " + commandPrefix2 + "unlock " + "Tron",
+                            False)
+
     _embedMessage.add_field(helpMessage, commandPrefix2 + "say title content channel",
-                            "Sends a message inside an embed to the specified channel\nEx: " + commandPrefix2 + "say Hello world " + ctx.channel.mention, False)
+                            "ADMIN ONLY - Sends a message inside an embed to the specified channel\nEx: " + commandPrefix2 + "say Hello world " + ctx.channel.mention, False)
     _embedMessage.add_field(helpMessage, commandPrefix2 + "setbedibotchannel",
-                            "Sets the channel which will be used for announcements\nWARNING: This clears the channel's history. Use with caution.", False)
-    _embedMessage.add_field(helpMessage, commandPrefix2 + "settings", "Displays the guild's settings", False)
-    _embedMessage.add_field(helpMessage, commandPrefix2 + "unlock role", "Sets send message permissions to True for specified role\nEx: " + commandPrefix2 + "unlock " + "Tron",
-                            False)
+                            "ADMIN ONLY - Sets the channel which will be used for announcements\nWARNING: This clears the channel's history. Use with caution.", False)
+    _embedMessage.add_field(helpMessage, commandPrefix2 + "settings", "ADMIN ONLY - Displays the guild's settings", False)
 
```

Comment:
If we're reorganizing $help, can these be contained in a separate embed that will check if they have admin and if they do, will send these messages off?

Reply:
I've made this change. Good idea.